packages/ui/src/Index.tsx

```diff
@@ -15,8 +16,10 @@ export function Index({projects}: IndexProps) {
       <div className={styles.header}>Projects</div>
       <div className={styles.list}>
         {projects.map(project => (
-          <a className={styles.element} href={`./${project.name}`}>
-            <div className={styles.title}>{project.name}</div>
+          <a className={styles.element} href={`./${project.fileName}`}>
+            <div className={styles.title}>
+              {project.name || project.fileName}
```

Comment:
It's not possible for `name` to not be defined here

Reply:
Sorry, I don't understand. Why do you say it is not possible? 
Do You mean not technically, but for other reasons?
Because it is working like that:
<img width="611" alt="image" src="https://user-images.githubusercontent.com/14844155/226970499-d9d9e685-9461-421c-b3f6-c9837e077dbe.png">